FindxDAQ.cmake

```diff
@@ -141,25 +157,25 @@ function(xdaq_import_lib name)
     set_property(
```

Comment:
Is there a reason why you do not use `set_properties()` for this series of properties assignments ? I think it would reduce the code length and be as (or more) readable as it is now.

Reply:
More exactly `set_target_properties()` :) I didn't know about it

Reply:
Oops...

Reply:
`set_target_properties()` wouldn't help because I need to set lists and lists cannot be created with the `set_target_properties()` command.